tests/functional/json/slate_lineup_config.py

```diff
@@ -1,15 +1,32 @@
 from app.models.slate_lineup_experiment import SlateLineupExperimentModel
 from app.models.slate_config import SlateConfigModel
 from app.models.slate_lineup_config import SlateLineupConfigModel
+from collections import Counter
 
 
 def test_slate_lineup_config():
     """
-    Test that all slates referenced in the slate_lineup config exist.
+    Test that slate and lineup configs are valid
     """
     SlateConfigModel.SLATE_CONFIGS_BY_ID = {s.id: s for s in SlateConfigModel.load_slate_configs()}
     SlateLineupConfigModel.SLATE_LINEUP_CONFIGS = SlateLineupConfigModel.load_slate_lineup_configs()
 
+    # Ensure GUIDs are unique
+    slate_ids = Counter(SlateConfigModel.SLATE_CONFIGS_BY_ID.keys())
+    for sid, count in slate_ids.items():
+        if count > 1:
+            raise ValueError(f"Slate GUID {sid} appears more than once in slate config")
+
+    lineup_ids = Counter([r.id for r in SlateLineupConfigModel.SLATE_LINEUP_CONFIGS])
+    for lid, count in lineup_ids.items():
+        if count > 1:
+            raise ValueError(f"SlateLineup GUID {lid} appears more than once in slate lineup config")
+
+    guid_overlap = set(lineup_ids.keys()).intersection(slate_ids.keys())
+    if guid_overlap:
+        raise ValueError(f"GUID is reused for slate and lineup: {guid_overlap}")
```

Comment:
There are faster ways to do this, but they either require an extra step to get the actual duplicated ID or aren't as clear to follow as this, so I'm happy with this solution.